lv_lgia/lks92to2020_convert.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+python ../grid_tools/ntv2_to_gtiff.py \
+    --source-crs EPSG:4661 \
+    --target-crs EPSG:10305 \
+    --copyright "Derived from work by Latvian Geospatial Information Agency. CC-BY 4.0"\
+    --description "LKS-92 (EPSG:4661) to LKS-2020 (EPSG:10305). Converted from LKS92to2020NTv2.gsb." \
+    --positive-longitude-shift-value west \
```

Comment:
Normally when converting from NTv2 to GTiff, we indicate east, as this is more intutive when inspecting the GeoTIFF. The script will take care of inverting the size of the longitude offset band when the source is NTv2

Reply:
I got the impression from the EPSG registration that the data was oriented in the west direction. I tested the grid against the supplied test coordinates and it seemed to work. Are you saying that it would have worked either way? I can change it if you think this isn't a smart way to make the grid.

Reply:
> Are you saying that it would have worked either way?

yes, both conventions are supported by PROJ, but it is nicer to use the positive=east convention to avoid the NTv2 oddity to further propagate

Reply:
Got it. I simply misunderstood the use of that option.